test/ComposerRequireCheckerTest/Cli/OptionsTest.php

```diff
@@ -63,9 +62,22 @@ public function testOptionsFileRepresentsDefaults(): void
         $fileContent = file_get_contents(__DIR__ . '/../../../data/config.dist.json');
         $this->assertNotFalse($fileContent);
 
-        $optionsFromFile = new Options(
-            json_decode($fileContent, true),
-        );
+        $optionsFromFile = new Options([
+            'symbol-whitelist'    => [],
+            'php-core-extensions' => [
+                'Core',
+                'date',
+                'json',
+                'hash',
+                'pcre',
+                'Phar',
+                'Reflection',
+                'SPL',
+                'random',
+                'standard',
+            ],
+            'scan-files'          => [],
+        ]);
 
         $this->assertEquals($options, $optionsFromFile);
     }
```

Comment:
`$fileContent` is no longer used in this test. We are now comparing `new Options()` with `new Options( $some_fixed_content )`. The file isn't used at all here any longer. Perhaps this test should be renamed or rewritten?

Reply:
Yep, renaming is a good idea